analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -62,25 +63,31 @@
 """
 
 
+class SkipHandler(SkipListHandler):
```

Comment:
This is a little bit strange for me that we have a `SkipListHandler` and a `SkipHandler` classes. Which one should I use?

Maybe we can handle these changes in the original `SkipListHandler` class.